code/game/gamemodes/eldritch_cult/eldritch_cult.dm

```diff
@@ -0,0 +1,69 @@
+/datum/game_mode/ecult
+	name = "eldritch cultist"
+	config_tag = "eldritch cultist"
+	report_type = "eldritch cultist"
+	antag_flag = ROLE_ECULT
+	false_report_weight = 5
+	protected_jobs = list("Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	restricted_jobs = list("AI", "Cyborg","Chaplain")
+	required_players = 0
+	required_enemies = 1
+	recommended_enemies = 4
+	reroll_friendly = 1
+	enemy_minimum_age = 0
```

Comment:
you sure about this one, chief?

this is the minimum number of rounds you need to play in order to qualify for being chosen for this antag role, right?

Reply:
I dont actually know, i copied what worked in traitor gamemode, ill have to look into it.